daprdocs/content/en/reference/api/configuration_api.md

```diff
@@ -165,18 +166,22 @@ None
 
 Code | Description
 ---- | -----------
-`204`  | Unsubscribe operation successful
+`200`  | Unsubscribe operation successful
 `400`  | Configuration store is missing or misconfigured or malformed request
 `500`  | Failed to unsubscribe to configuration changes
 
 #### Response Body
 
-None
+```
+{
+    "ok" : true
+}
+```
 
 ### Example
 
 ```shell
-curl -X GET 'http://localhost:3500/v1.0-alpha1/configuration/bf3aa454-312d-403c-af95-6dec65058fa2/unsubscribe' 
+curl -X GET 'http://localhost:3500/v1.0-alpha1/configuration/myconfigstore/bf3aa454-312d-403c-af95-6dec65058fa2/unsubscribe' 
```

Comment:
Either suggest changing to all `myconfigstore` or change this example to `mystore`

Reply:
I fixed the above to make consistent

Reply:
thanks!